# Try and Except statements are used to catch and handle the Exceptions.

# Example - 

try:
    a = 10/0
except ZeroDivisionError:
    print("No exception raised")